### ISSUE TITLE
Dropdown Menus Switch Direction

### ISSUE THREAD

**Issue by Seercat3160** [MegaAntiCheat/MegaAntiCheat-UI#61]
## Description of the Problem

Dropdown menus (such as the one to change a user's local verdict or the right-click menu to votekick or copy ID) sometimes flicker between going up and down from the anchor (verdict indicator or user right-clicked). This seems to occur more when interacting with users near the bottom of the window, such as unassigned or spectators. It's possible that they only switch directions when the mouse is moved onto them, but I'm not sure.

This makes it near-impossible to actually click anything within the menu when this happens.

Unfortunately I have no screenshots or video to show what I mean, let me know if any clarification or further information is needed.

## Usage Details

Browser: Steam Overlay so Chromium of some variety (though I simply don't use the webui in other browsers enough to know if it's browser-specific)

## How to Reproduce

I have no specific reproduction steps aside from right-clicking on users and trying to click something on the right-click menu until it happens.

**Comment by Bash-09** [comment]
I've also run into this a lot before, particularly when I ran Firefox in the corner of a second monitor (so it's only half the height on the monitor and so a regular 24-player server list usually gets close to the bottom of the window). Trying to right click and use the context menu of a player who is near the bottom of the window (I assume in a position where opening the context menu would escape the bounds of the visible area in the browser) would make it jitter and be unusable.

Since then I've just made the window taller so there aren't any players close enough to the bottom to trigger it.
